WordPressComKit/Me.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public class Me {
```

Comment:
Also, I think every model should have documentation, even if it's just copying the REST API doc (although many of those need improvement)